src/sentry/monitors/validators.py

```diff
@@ -98,6 +99,7 @@ class ConfigValidator(serializers.Serializer):
         default=None,
         help_text="How long (in minutes) is the checkin allowed to run for in CheckInStatus.IN_PROGRESS before it is considered failed.",
         min_value=1,
+        max_value=MAX_TIMEOUT,
```

Comment:
This change is separate from the split up. Can we actually make that in a second PR?

Reply:
https://github.com/getsentry/sentry/pull/53175